spec/models/role_spec.rb

```diff
@@ -4,8 +4,8 @@
 
 describe Role do
   let(:conference) { create(:conference) }
-  let!(:organizer_role) { Role.find_by(name: 'organizer', resource: conference) }
-  let!(:cfp_role) { Role.find_by(name: 'cfp', resource: conference) }
+  let!(:organizer_role) { Role.find_or_create_by(name: 'organizer', resource: conference) }
+  let!(:cfp_role) { Role.find_or_create_by(name: 'cfp', resource: conference) }
```

Comment:
Why isn't this a factory?

Reply:
Because there should be there, but it seems that after the update they are not anymore. There are that many things in those tests that need to be changed, that I would focus on finishing the Rails update (there is still a long way to achieve this and there is a security issue) and refactor/rewrite the tests&code afterwards